package.json

```diff
@@ -58,41 +58,41 @@
     "@openmrs/esm-framework": "next",
     "@openmrs/esm-styleguide": "next",
     "@swc/cli": "^0.1.57",
-    "@swc/core": "^1.3.6",
-    "@swc/jest": "^0.2.23",
+    "@swc/core": "^1.3.22",
+    "@swc/jest": "^0.2.24",
     "@testing-library/dom": "^8.19.0",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.4.3",
-    "@types/jest": "^28.0.0",
-    "@types/react": "^18.0.21",
-    "@types/react-dom": "^18.0.6",
+    "@types/jest": "^29.2.4",
+    "@types/react": "^18.0.26",
+    "@types/react-dom": "^18.0.9",
     "@types/react-router": "^5.1.19",
     "@types/react-router-dom": "^5.3.3",
-    "@types/webpack-env": "^1.16.0",
-    "@typescript-eslint/parser": "^5.40.0",
-    "concurrently": "^7.4.0",
+    "@types/webpack-env": "^1.18.0",
+    "@typescript-eslint/parser": "^5.46.1",
+    "concurrently": "^7.6.0",
     "css-loader": "^6.7.2",
-    "eslint": "^8.25.0",
+    "eslint": "^8.29.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-config-ts-react-important-stuff": "^3.0.0",
     "eslint-plugin-prettier": "^4.2.1",
     "husky": "^8.0.2",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^28.0.0",
-    "jest-cli": "^28.0.0",
-    "jest-environment-jsdom": "^28.0.0",
+    "jest": "^29.3.1",
+    "jest-cli": "^29.3.1",
+    "jest-environment-jsdom": "^29.3.1",
     "openmrs": "next",
-    "prettier": "^2.7.1",
+    "prettier": "^2.8.1",
     "pretty-quick": "^3.1.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-i18next": "^11.18.6",
-    "react-router-dom": "^6.4.2",
-    "rxjs": "^6.5.4",
+    "react-i18next": "^12.1.1",
+    "react-router-dom": "^6.4.5",
+    "rxjs": "^7.6.0",
```

Comment:
rxjs and react-i18next should be updated, but that requires more coordination across repos (so we can bump the peer deps everywhere). I'm not sure it's a good idea to bump the dev versions without also bumping the peerdeps...